src/rstrtmgr.cpp

```diff
@@ -0,0 +1,86 @@
+/* ****************************************************************************
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+*
+*
+* This file is part of Microsoft R Host.
+*
+* Microsoft R Host is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Microsoft R Host is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Microsoft R Host.  If not, see <http://www.gnu.org/licenses/>.
+*
+* ***************************************************************************/
+
+#include "rstrtmgr.h"
+#include "Rapi.h"
+#include <RestartManager.h>
+
+namespace rhost {
+    namespace util {
+#define PACKAGE_LOCK_STATE_UNLOCKED        "unlocked"
```

Comment:
#define should only be used for macros (copy/pasted stuff from R headers excepted). These are best declared as `const char ...[] = ...`.